apps/web/src/app/reconnect/page.module.css

```
@@ -0,0 +1,163 @@
+  border-radius: 500px;
+  background: #fee500;
+}
+
+.srOnly {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.3.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)